ui/pages/index.tsx

```diff
@@ -53,46 +120,91 @@ export default function Dashboard() {
-                  href="https://github.com/corshatech/cast/wiki/Activating-Optional-Features#feodo-banlist-data"
-                >
-                  <EnablementChip
-                    label='IP Banlist Data'
-                    tooltipEnabled='FEODOTracker data has been loaded.'
-                    tooltipDisabled='No FEODOTracker ddata has been found. Check the CAST Wiki for details.'
-                    enabled={enablemenets.feodoEnabled}
-                  />
-                </a>
-              </>}
+              <Typography variant="h1" className='md:text-5xl xl:text-7xl'>
+                Ongoing Results
+              </Typography>
+              <div className='flex flex-col md:flex-row space-y-5 md:space-y-0 md:items-center md:space-x-3 md:ml-1 md:pt-6 xl:pt-8'>
+                {summary ? (
+                  <>
+                    <Typography variant="h4" className='md:text-2xl xl:text-3xl'>
+                      {summaryTitle}
+                    </Typography>
+                    <Typography style={{ color: theme.palette.text.secondary }} className='xl:text-xl font-semibold'>
+                      {summary?.scansPassed || 0} passed
+                    </Typography>
+                  </>
+                ) : (
+                  <>
+                    <div className={'h-9 w-60 rounded-lg bg-slate-50 bg-slate-400/10 animate-pulse'} />
+                    <div className={'h-7 w-20 rounded-lg bg-slate-50 bg-slate-400/10 animate-pulse'} />
+                  </>
+                )}
+              </div>
             </div>
 
-            <div className="flex flex-col 4K:grid 4K:grid-cols-2 space-y-12 4K:space-y-0 4K:gap-4 mb-12">
-              {analyses.length > 0 ? (
-                analyses.map((a) => <AnalysisCard {...a} key={a.id} />)
-              ) : (
-                <AnalysisGridLoading />
-              )}
-            </div>
+            {isLoading && analyses.length === 0 && !error ? (
+              <div className="absolute w-full h-full z-50 bg-slate-400/10 animate-pulse"/>
+            ) : (
+              <Tabs defaultValue="critical">
+                <TabsList className='w-full h-14 overflow-x-scroll grid grid-flow-col justify-stretch'>
+                  <TabsTrigger className='h-full' value="critical">Critical {findingsChip('critical')}</TabsTrigger>
+                  <TabsTrigger className='h-full' value="high">High {findingsChip('high')}</TabsTrigger>
+                  <TabsTrigger className='h-full' value="medium">Medium {findingsChip('medium')}</TabsTrigger>
+                  <TabsTrigger className='h-full' value="low">Low {findingsChip('low')}</TabsTrigger>
+                  <TabsTrigger className='h-full' value="none">None {findingsChip('none')}</TabsTrigger>
+                  <TabsTrigger className='h-full' value="status">Status</TabsTrigger>
+                  <TabsTrigger className='h-full' value="settings">Settings</TabsTrigger>
+                </TabsList>
```

Comment:
Let's use `items-stretch` instead of `h-full` on each item.
```suggestion
                <TabsList className='w-full h-14 overflow-x-scroll grid grid-flow-col items-stretch justify-stretch'>
                  <TabsTrigger value="critical">Critical {findingsChip('critical')}</TabsTrigger>
                  <TabsTrigger value="high">High {findingsChip('high')}</TabsTrigger>
                  <TabsTrigger value="medium">Medium {findingsChip('medium')}</TabsTrigger>
                  <TabsTrigger value="low">Low {findingsChip('low')}</TabsTrigger>
                  <TabsTrigger value="none">None {findingsChip('none')}</TabsTrigger>
                  <TabsTrigger value="status">Status</TabsTrigger>
                  <TabsTrigger value="settings">Settings</TabsTrigger>
                </TabsList>
```